README.rst

```diff
@@ -141,6 +141,35 @@ To change the MFEs logos from the default to your own logos, override the corres
         LOGO_WHITE_URL=<URL>/logo-white.svg
         FAVICON_URL=<URL>/favicon.ico
 
+To install custom components for the MFEs, such as the `header <https://github.com/openedx/frontend-component-header>`_ and `footer <https://github.com/openedx/frontend-component-footer>`_, override the components by adding a patch to ``mfe-dockerfile-post-npm-install`` in your plugin:
+::
+
+    patches:
+        ...
+        mfe-dockerfile-post-npm-install |
+            # npm package
+            RUN npm install '@edx/frontend-component-header@npm:@edx/frontend-component-header-edx@latest'
+            # git repository
+            RUN npm install '@edx/frontend-component-footer@git+https://github.com/edx/frontend-component-header-edx.git'
+
+The same applies to installing a custom `brand <https://github.com/openedx/brand-openedx>`_ package:
+::
+
+    patches:
+        ...
+        mfe-dockerfile-post-npm-install |
+            RUN npm install '@edx/brand@git+https://github.com/edx/brand-edx.org.git'
+
+In case you need to install components from a private NPM registry, you can append the ``--registry`` option to your install statement or add a ``npm config set`` command to the plugin.
```

Comment:
Would you mind adding a different title to this section, such as "Installing from a private npm registry", or something similar?

Reply:
Sure, will do.